BrainPortal/app/views/tool_configs/_tool_configs_table.html.erb

```diff
@@ -104,6 +104,13 @@
         end
       }
 
+    t.column("Container Index", :container_index_location,
+    ) { |tc|
+        if tc.container_index_location.present?
```

Comment:
the whole if block can be replaced by

```ruby
tc.container_index_location
```

there is no need to check for presence/blankness etc.